src/TwilioVideo.android.js

```diff
@@ -15,8 +15,8 @@ import {
   NativeModules,
   findNodeHandle,
 } from 'react-native';
-import React, { PropTypes, Component } from 'react';
-
+import React, { Component } from 'react';
```

Comment:
What version of react removed the `PropTypes` ? We need to add a constraint to make sure it's compatible